File: beecrowd/1/1011/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
    var R float64
    fmt.Scanf("%v\n", &R)
    fmt.Printf("VOLUME = %.3f\n",(4.0/3) * 3.14159 * math.Pow(R, 3) )
}

File: beecrowd/1/1011/main.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

int main() {
    double R;
    std::cin >> R;
    std::cout << std::fixed << std::setprecision(3)
        << "VOLUME = " << (4.0/3) * 3.14159 * std::pow(R, 3) << std::endl;
    return 0;
}
